README.md

```diff
@@ -91,41 +91,42 @@ const coordinates = h3.h3SetToMultiPolygon(hexagons, true);
     * [.h3IsPentagon(h3Index)](#module_h3.h3IsPentagon) ⇒ <code>boolean</code>
     * [.h3IsResClassIII(h3Index)](#module_h3.h3IsResClassIII) ⇒ <code>boolean</code>
     * [.h3GetBaseCell(h3Index)](#module_h3.h3GetBaseCell) ⇒ <code>number</code>
-    * [.h3GetFaces(h3Index)](#module_h3.h3GetFaces) ⇒ <code>Array.&lt;number&gt;</code>
+    * [.h3GetFaces(h3Index)](#module_h3.h3GetFaces) ⇒ <code>[ &#x27;Array&#x27; ].&lt;number&gt;</code>
```

Comment:
This suggests you have some dependency mismatch, maybe from using `npm` instead of `yarn`?